### PR TITLE
Exposing OperationTimeout property's setter on all clients

### PR DESCRIPTION
**Pull request by nemakam**
## Description
Exposing OperationTimeout property's setter on all clients

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](./CONTRIBUTING.md).**
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR.
- [x] The pull request does not introduce breaking changes (unless a major version change occurs in the assembly and module).
- [x] If applicable, the public code is properly documented.
- [x] Pull request includes test coverage for the included changes.
- [x] The code builds without any errors.

### CODE REVIEW

File: src/Microsoft.Azure.ServiceBus/Core/MessageReceiver.cs [hunk 1]
```diff
@@ -202,6 +202,14 @@ internal set
         public virtual string Path { get; private set; }
 
         /// <summary>
+        /// Duration after which individual operations will timeout.
```
Comment (SeanFeldman) [inline src/Microsoft.Azure.ServiceBus/Core/MessageReceiver.cs]: Would it be better to use `<inheritDoc/>`?

File: src/Microsoft.Azure.ServiceBus/Core/MessageSender.cs [hunk 2]
```diff
@@ -111,6 +111,15 @@ public class MessageSender : ClientEntity, IMessageSender
         /// </summary>
         public virtual string Path { get; private set; }
 
+        /// <summary>
```
Comment (SeanFeldman) [inline src/Microsoft.Azure.ServiceBus/Core/MessageSender.cs]: Would it be better to use `<inheritDoc/>`?

File: src/Microsoft.Azure.ServiceBus/QueueClient.cs [hunk 3]
```diff
@@ -126,6 +126,15 @@ public QueueClient(string connectionString, string entityPath, ReceiveMode recei
         public ReceiveMode ReceiveMode { get; }
 
         /// <summary>
+        /// Duration after which individual operations will timeout.
```
Comment (SeanFeldman) [inline src/Microsoft.Azure.ServiceBus/QueueClient.cs]: Would it be better to use `<inheritDoc/>`?

File: src/Microsoft.Azure.ServiceBus/SessionClient.cs [hunk 4]
```diff
@@ -141,6 +141,15 @@ public sealed class SessionClient : ClientEntity, ISessionClient
         /// </summary>
         public string EntityPath { get; }
 
+        /// <summary>
```
Comment (SeanFeldman) [inline src/Microsoft.Azure.ServiceBus/SessionClient.cs]: Would it be better to use `<inheritDoc/>`?

File: src/Microsoft.Azure.ServiceBus/SubscriptionClient.cs [hunk 5]
```diff
@@ -140,6 +140,15 @@ public SubscriptionClient(string connectionString, string topicPath, string subs
         public ReceiveMode ReceiveMode { get; }
 
         /// <summary>
+        /// Duration after which individual operations will timeout.
```
Comment (SeanFeldman) [inline src/Microsoft.Azure.ServiceBus/SubscriptionClient.cs]: Would it be better to use `<inheritDoc/>`?

File: src/Microsoft.Azure.ServiceBus/TopicClient.cs [hunk 6]
```diff
@@ -91,6 +91,15 @@ public TopicClient(string connectionString, string entityPath, RetryPolicy retry
         public string TopicName { get; }
 
         /// <summary>
+        /// Duration after which individual operations will timeout.
```
Comment (SeanFeldman) [inline src/Microsoft.Azure.ServiceBus/TopicClient.cs]: Would it be better to use `<inheritDoc/>`?

File: src/Microsoft.Azure.ServiceBus/Core/MessageReceiver.cs (hunk 1 above)
Comment (nemakam) [inline src/Microsoft.Azure.ServiceBus/Core/MessageReceiver.cs]: It looks like this tag isn't natively supported by C#. Do I need additional libraries to get this working? I tried using the tag and VS intellisense doesn't show the comment. Am I missing something?

**Comment by SeanFeldman** [comment]
It does 😞
Please ignore my comment.

**Pull request merged**